Sources/Private/Model/Layers/LayerModel.swift

```diff
@@ -88,7 +88,7 @@ class LayerModel: Codable {
   required init(from decoder: Decoder) throws {
     let container = try decoder.container(keyedBy: LayerModel.CodingKeys.self)
     name = try container.decodeIfPresent(String.self, forKey: .name) ?? "Layer"
-    index = try container.decodeIfPresent(Int.self, forKey: .index) ?? 0
+    index = try container.decodeIfPresent(Int.self, forKey: .index) ?? UUID().hashValue
```

Comment:
The experimental rendering engine requires that every layer has a unique `index`, so I changed the fallback from `0` to an `Int` derived from a `UUID`.

Reply:
Seems like a kinda roundabout way to get to an `Int.random(…)` — do you have a reason to believe you'll get a better result from UUID?

Reply:
`Int.random` sounds better, `UUID` was just the first thing I thought of